tests/dummy/app/templates/docs/configuration.md

```diff
@@ -12,7 +12,7 @@ However, there are a few things worth noting if you're already familiar with Typ
 
 2. Closely related to the previous point: any changes you do make to `outDir` won't have any effect on how _Ember_ builds your application—we have to pipe everything into Ember CLI via [broccoli], so we override that. In general, everything else works just as you'd expect, though!
 
-3. By default, we target the highest stable version of JavaScript available in the TypeScript compiler, so that you may ship anything from that very code without further modification to browsers that support it all the way back to ES3, in line with the Babel configuration in your app's `config/targets.js`. You can set this target to whatever is appropriate for your application, but we _strongly_ encourage you to leave it set to the highest stable version of JavaScript if you are developing an addon, so that consumers of your addon have full flexibility in this regard.
```

Comment:
I don't believe this is true in all cases. If you don't specify a `lib`, language version is inferred from `target`

example

```ts
const p = new Promise((res, reject) => {})
```
won't type-check with `target` of `es5` because of the missing promise constructor https://typescript-play.js.org/?target=1#code/MYewdgzgLgBADjAvDMBTA7jACgJxAWwEsJUAKUnVCAGhkoCtVgoBKJAPhgG8BfFgbgBQQA

Reply:
Ah, excellent point. I'll clarify that we specify the latest spec version for that reason, and let Babel and/or polyfills do the rest.